### ISSUE TITLE
Sorting for the NCL Index

### ISSUE THREAD

**Issue by kafitzgerald** [NCAR/geocat-applications#119]
It might be worth exploring using something like [DataTables](https://datatables.net/) to allow for sorting and better interactions with tables especially for things like the NCL Index. 

It looks like someone wrote a [sphinx-datatables](https://github.com/sharm294/sphinx-datatables) package, but it seems a bit like a side project and it may not be all that hard to use DataTables more directly.

**Comment by kafitzgerald** [comment]
Actually, ITables looks like it might be a really nice way to go for this: https://mwouts.github.io/itables

I have a local demo working.

**Comment by kafitzgerald** [comment]
Any initial thoughts on this?  @anissa111, I think you took a quick look.

There are a few things I'd like to update with respect to the formatting, but I don't think the defaults here are much worse than the current table and these should be relatively easy to address.  Things that come to mind are and I'd like to update before merging are:
* Text alignment
* Column size

I think the main concern I have with implementing ITables for the NCL Index as I demoed here is that in order to get the links and code formatting to work I needed to add HTML to the CSV file.  This seems potentially a bit error prone and painful to edit, but on the flip side we already have some formatting syntax in there right now.

**Comment by anissa111** [comment]
Yeah! I think it's a good idea. I didn't try to get a local demo going, but I'd be interested to see what yours ended up looking like locally.

I wish we had something a bit less clunky than the CSV file, but I don't think adding some html links is too much additional mess to avoid doing it

**Comment by kafitzgerald** [comment]
Sorry, I should have added links as well.

Demo NCL Index page from the PR preview: https://ncar.github.io/geocat-applications/_preview/210/ncl/ncl_index/ncl_index.html
Code diffs for the initial demo: https://github.com/NCAR/geocat-applications/pull/210/files

**Comment by anissa111** [comment]
Ah yeah, I saw that! 

I thought that you maybe had more set up locally (I possibly got that impression from https://github.com/NCAR/geocat-applications/pull/210#issuecomment-2947815708)
